README.md

```diff
@@ -54,6 +54,10 @@ helm install ls-wg-system-metrics cribl/logstream-workergroup \
   -n logstream
 ```
 
+* Running Distributed on a Free License
+
+If you are not specifying a license in your install, and you're looking to run distributed (free license allows one workergroup), you'll need to go into the user interface for logstream and accept the free license. If you specify the groups option, the master will be configured as a distributed master. If you don't, it will be configured as a standalone instance, and you'll need to manually configure it via the UI..
```

Comment:
@stevelitras This says (if I'm reading it correctly): (1) you want to run distributed > (2) you need to configure licensing in LogStream's UI > (3) you need to specify `groups` ¿back in the Helm chart? > (4) otherwise you'll get single-instance, and (5) you'll need to select `Distributed` in LogStream's UI.

Can we rearrange it to list (a) first all Helm setup, and then (b) all LogStream UI steps together? 

Also, can we clarify whether any of this applies to users who _want_ a single-instance deployment –  do they read to read this, or should they just ignore it? They'll also need to accept the free license, but we cover that [here](https://github.com/criblio/helm-charts/pull/32/files#diff-5a19ad0044b760f9e09b807a1e16bd648b9f160c471195fb479ad7556cf532c9R94)